templates/base/head_navbar.tmpl

```diff
@@ -18,7 +18,7 @@
 				</span>
 			</a>
 			{{end}}
-			<div class="ui icon button mobile-only" id="navbar-expand-toggle">
+			<div class="ui icon button mobile-only" id="navbar-expand-toggle" tabindex="0" role="button">
 				{{svg "octicon-three-bars"}}
 			</div>
```

Comment:
Shouldn't the following produce the same result but with even better a11y support?
```suggestion
			<button class="ui icon button mobile-only" id="navbar-expand-toggle">
				{{svg "octicon-three-bars"}}
			</button>
```

Reply:
Yes, a real button is better than `role=button`.